browser/src/view/components/Button/Button.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './Button.scss';
+
+export const Button = ({ layout, label, Icon, type }) => {
+  return (
+    <button
+      className={`${
+        type === ButtonAttributes.Type.destructive
+          ? 'btn-destructive'
+          : type === ButtonAttributes.Type.primary
+          ? 'btn-primary'
+          : 'btn-secondary'
```

Comment:
Why is secondary the default?

Reply:
I think I simply just messed up the ordering haha